### PR TITLE
Commit pipelinerun tab

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/HACBS-828

## Description

Adds PipelineRun tab to commits details

## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2022-09-07 at 8 06 56 PM
Screenshot 2022-09-07 at 8 07 22 PM

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: config/remotePlugin.js [hunk 1]
```diff
@@ -97,6 +98,32 @@ const hacbs = {
...
+      type: 'console.page/route',
+      properties: {
+        path: '/app-studio/:appName/commit/:commitName',
+        exact: true,
+        component: {
+          $codeRef: 'CommitsPage',
+        },
+      },
+      flags: {
+        required: ['HACBS'],
```
Comment (christianvogt) [inline config/remotePlugin.js:111]: Please ensure hacbs pages also include the `SIGNUP` flag

File: src/hacbs/components/Commits/tabs/CommitsPLRTab.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        Row={PipelineRunListRow}
+        getRowProps={(obj: PipelineRunKind) => ({
+          id: obj.metadata.name,
+        })}
+      />
+    </>
+  );
+};
+
+export default CommitsPLRTab;
```
Comment (christianvogt) [inline src/hacbs/components/Commits/tabs/CommitsPLRTab.tsx:78]: You can delete this file.

File: src/hacbs/components/Commits/tabs/CommitsPipelineRunTab.tsx [hunk 3]
```diff
@@ -1,13 +1,81 @@
...
-  return <>PipelineRun of {getCommitDisplayName(commit)}</>;
+  pipelineRuns?.sort(
+    (app1, app2) =>
+      +new Date(app2.metadata.creationTimestamp) - +new Date(app1.metadata.creationTimestamp),
+  );
+
+  return (
+    <>
+      <Title headingLevel="h4" className="pf-c-title pf-u-mt-lg pf-u-mb-lg">
+        Pipelineruns
```
Comment (christianvogt) [inline src/hacbs/components/Commits/tabs/CommitsPipelineRunTab.tsx:65]: `Pipeline runs`

File: config/remotePlugin.js (hunk 1 above)
Comment (abhinandan13jan) [reply config/remotePlugin.js]: updated

File: src/hacbs/components/Commits/tabs/CommitsPLRTab.tsx (hunk 2 above)
Comment (abhinandan13jan) [reply src/hacbs/components/Commits/tabs/CommitsPLRTab.tsx]: deleted

File: src/hacbs/components/Commits/tabs/CommitsPipelineRunTab.tsx (hunk 3 above)
Comment (abhinandan13jan) [reply src/hacbs/components/Commits/tabs/CommitsPipelineRunTab.tsx]: updated

**Comment by abhinandan13jan** [comment]
Screenshot 2022-09-16 at 7 44 43 PM

**Pull request merged**